.travis.yml

```diff
@@ -13,6 +13,7 @@ before_script:
 
 matrix:
   include:
+    # These environments are executed on push and cron events
```

Comment:
They're also executed on pull requests? Maybe we can just say "These environments are always executed."?

Reply:
Yes I am influenced by Gitlab that does not make the distinction at this level. Formely you always push to a reference, pull_request or not. It is better to say always, or only in cron jobs.